query-connector/README.md

```diff
@@ -8,33 +8,6 @@ The DIBBs Query Connector app offers a REST API for searching for a patient and
 
 The Query Connector app can be run using Docker (or any other OCI container runtime e.g., Podman), or directly from the Node.js source code.
 
-#### Obtaining an eRSD API Key
```

Comment:
moved this chunk to after the install instructions for information hierarchy reasons